wordfreq/_chinese_mapping.py

```diff
@@ -0,0 +1,3275 @@
+SIMPLIFIED_MAP = {13423: '㑔',
```

Comment:
125 ms is in the starting-to-add-up zone, at least (sigh) while we still have stateless calculators.